### PR TITLE
Update WikiLookup.json with UESP.net Wiki

### PR DESCRIPTION
**Pull request by DaveHumphrey**

### CODE REVIEW

File: WikiLookup.json [hunk 1]
```diff
@@ -575,5 +575,46 @@
...
+		"name": "Unofficial Elder Scrolls Pages",
+		"homepage": "https://en.uesp.net	",
+		"lang": "en",
+		"api": "https://en.uesp.net/w/api.php",
+		"companies": [
+		    "Bethesda Softwarks",
+			"Zenimax Online Studios"
+		],
+		"games": [
+		    "Arena",
```
Comment (prodigion) [inline WikiLookup.json:589]: Please use consistent spacing throughout. Lines should only use tabs.

File: WikiLookup.json [hunk 2]
```diff
@@ -575,5 +575,46 @@
...
+			"Call to Arms",
+			"Dawnstar",
+			"Legends",
+			"Oblivion Mobile",
+			"Shadowkey",
+			"Skyrim Pinball",
+			"Stormhold"
+		],
+		"genres": [
+		    "Role Playing"
```
Comment (prodigion) [inline WikiLookup.json:607]: As with `companies`, `genres` shouldn't be included.

File: WikiLookup.json [hunk 3]
```diff
@@ -575,5 +575,46 @@
...
 			"Squadron 42"
 		]
+	},
+		{
+		"name": "Unofficial Elder Scrolls Pages",
+		"homepage": "https://en.uesp.net	",
+		"lang": "en",
+		"api": "https://en.uesp.net/w/api.php",
+		"companies": [
+		    "Bethesda Softwarks",
```
Comment (prodigion) [inline WikiLookup.json:585]: As this wiki only covers a subset of the games Bethesda/Zenimax have created, it shouldn't include a `companies` entry.

File: WikiLookup.json [hunk 4]
```diff
@@ -575,5 +575,46 @@
...
+			"Skyrim Pinball",
+			"Stormhold"
+		],
+		"genres": [
+		    "Role Playing"
+		],
+		"series": [
+		    "The Elder Scrolls"
+		],
+		"systems": [
```
Comment (prodigion) [inline WikiLookup.json:612]: As with `companies`, `systems` shouldn't be included.

File: WikiLookup.json (hunk 1 above)
Comment (prodigion) [reply WikiLookup.json:589]: Also, please use the complete names for the games, such as `The Elder Scrolls: Arena`.

File: WikiLookup.json (hunk 2 above)
Comment (DaveHumphrey) [reply WikiLookup.json:607]: Note: As someone unfamiliar with the project it would be good to include all this information in the Readme.

Comment (prodigion) [reply WikiLookup.json:607]: Thanks, I'll update it!

**Comment by prodigion** [comment]
Thanks for the feedback! I've added the following clarification. Do you feel this is enough?

`For example, a wiki that covers the Need for Speed series of racing games would include a full games list, but would not include a racing entry in the genres section.`

**Pull request closed without merging**